de_dresden.csv

```diff
@@ -24,3 +24,4 @@ Dresden,2020-03-28,12:00,312
 Dresden,2020-03-29,12:00,320
 Dresden,2020-03-30,12:00,339
 Dresden,2020-03-31,12:00,358
+Dresden,2020-03-31,12:00,378
```

Comment:
s/Dresden,2020-03-31,12:00,378/Dresden,2020-04-01,12:00,378/

Reply:
I copy and pasted the data from the log file your PR produced.